src/Models/ShallowWaterModels/shallow_water_advection_operators.jl

```diff
@@ -77,13 +97,19 @@ a velocity field U = (u, v), ∇·(Uc),
 
 which will end up at the location `ccc`.
 """
-@inline function div_Uc(i, j, k, grid, advection, solution, c)
-    1/Vᶜᶜᶜ(i, j, k, grid) * (δxᶜᵃᵃ(i, j, k, grid, transport_tracer_flux_x, advection, solution.uh, solution.h, c) +        
-                             δyᵃᶜᵃ(i, j, k, grid, transport_tracer_flux_y, advection, solution.vh, solution.h, c))
+
+@inline function div_Uc(i, j, k, grid, advection, solution, c, formulation)
+    1/Vᶜᶜᶜ(i, j, k, grid) * (δxᶜᵃᵃ(i, j, k, grid, transport_tracer_flux_x, advection, solution[1], solution.h, c) +        
```

Comment:
Do we want to change this `V` and the one just below it as well?

Reply:
I'd say yes, we want to eliminate dz from all terms.